Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -1748,6 +1748,24 @@ - (IBAction)miniFormButtonTapped:(id)sender {
     NSMutableArray *steps = [NSMutableArray new];
     
     {
+        {
+            ORKFormStep *step = [[ORKFormStep alloc] initWithIdentifier:@"scale_form_00" title:@"Optional Form Items" text:@"Optional form with no required items and a default scale value"];
```

Comment:
Sorry, this was my mistake, it should read `Optional form with a required scale item with a default value`.

And add extra closing and opening bracket after step so it indents at the same levels as the next form steps.